openbb_platform/core/openbb_core/provider/standard_models/options_snapshots.py

```diff
@@ -20,56 +20,60 @@ class OptionsSnapshotsQueryParams(QueryParams):
 class OptionsSnapshotsData(Data):
     """Options Snapshots Data."""
 
-    underlying_symbol: str = Field(description="Ticker symbol of the underlying asset.")
-    contract_symbol: str = Field(description="Symbol of the options contract.")
-    expiration: dateType = Field(description="Expiration date of the options contract.")
-    dte: Optional[int] = Field(
-        default=None,
+    underlying_symbol: List[str] = Field(
+        description="Ticker symbol of the underlying asset."
+    )
+    contract_symbol: List[str] = Field(description="Symbol of the options contract.")
+    expiration: List[dateType] = Field(
+        description="Expiration date of the options contract."
+    )
+    dte: List[Union[int, None]] = Field(
+        default_factory=list,
         description="Number of days to expiration of the options contract.",
     )
-    strike: float = Field(
+    strike: List[float] = Field(
         description="Strike price of the options contract.",
         json_schema_extra={"x-unit_measurement": "currency"},
     )
-    option_type: str = Field(description="The type of option.")
-    volume: Optional[int] = Field(
-        default=None,
+    option_type: List[str] = Field(description="The type of option.")
+    volume: List[Union[int, None]] = Field(
+        default_factory=list,
         description=DATA_DESCRIPTIONS.get("volume", ""),
     )
-    open_interest: Optional[int] = Field(
-        default=None,
+    open_interest: List[Union[int, None]] = Field(
```

Comment:
Altough the `reference.json` seems to be well generated (default should be `null`, I believe) this syntax feels weird to me.
Should this be `Optional[List[int]` (and then defaults to `None`)? If we look at the 2nd part of the union it doesn't make sense: `List[None]` ([None,None, None]) - it's possible but odd